src/sdk/controllers/UserAuthorization.ts

```diff
@@ -100,4 +100,15 @@ export class UserAuthorization extends UserBase implements IUserAuthentication {
             privateKey,
         });
     }
+    async saveEthereumKey(masterPassword: string, options: ICreateAccountOptions): Promise<void> {
+        const res = await options.keyFromPasswordFn(masterPassword);
+
+        const privateKey = res.privateKey;
+
+        console.log('privatekey', privateKey);
+        await this.keyManager.storeKey({
+            level: KeyManagerLevel.ETHEREUM_KEY,
+            privateKey,
+        });
```

Comment:
@theblockstalk here i creates the private key how i can fetch the private key to perform transactions? i am bit confused about this

Reply:
lets not use the KeyManager for this. as it is outside the Pangea functions. Lets just make a key manager in Tonomy ID

Reply:
the key type `PrivateKey` used in our library is built for Antelope transaction signing. We need to convert this key type (which uses the underlying `curve` library I believe) into a format that we can send into the `web3` libraries for transaction signing